fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/join/JoinCommute.java

```diff
@@ -93,7 +93,7 @@ public static boolean check(SwapType swapType, LogicalJoin<GroupPlan, GroupPlan>
         if (swapType == SwapType.LEFT_ZIG_ZAG) {
             double leftRows = join.left().getGroup().getStatistics().getRowCount();
             double rightRows = join.right().getGroup().getStatistics().getRowCount();
-            return leftRows < rightRows && isZigZagJoin(join);
+            return leftRows <= rightRows && isZigZagJoin(join);
```

Comment:
why change to LE?

Reply:
initial planA: T1 join T2
if T1.rows = T2.rows, we need candidate planB: T2 join T1.
and then connectivity can be used to choose which plan is better